tests/HUBViewControllerFactoryTests.m

```diff
@@ -103,6 +103,30 @@ - (void)testCreatingViewControllerForInvalidViewURIReturnsNil
     XCTAssertNil([self.manager.viewControllerFactory createViewControllerForViewURI:viewURI]);
 }
 
+- (void)testCreatingViewControllerWithoutFeatureRegistration
+{
+    NSURL * const viewURI = [NSURL URLWithString:@"spotify:hub:framework"];
+    HUBContentOperationMock * const contentOperation = [HUBContentOperationMock new];
+    
+    __block BOOL contentOperationCalled = NO;
+    
+    contentOperation.contentLoadingBlock = ^(id<HUBViewModelBuilder> viewModelBuilder) {
+        contentOperationCalled = YES;
+        return YES;
+    };
+    
+    UIViewController<HUBViewController> * const viewController = [self.manager.viewControllerFactory createViewControllerForViewURI:viewURI
+                                                                                                                  contentOperations:@[contentOperation]
+                                                                                                                  featureIdentifier:@"identifier"
+                                                                                                                       featureTitle:@"Title"];
+    
+    [viewController viewWillAppear:NO];
+    
+    XCTAssertTrue(contentOperationCalled);
+    XCTAssertEqualObjects(viewController.featureIdentifier, @"identifier");
+    XCTAssertEqualObjects(viewController.navigationItem.title, @"Title");
```

Comment:
What about checking the feature registry for a registration for the provided URI?